libnmstate/netapplier.py

```diff
@@ -36,6 +36,7 @@
 from libnmstate.error import NmstateLibnmError
```

Comment:
The times in the commit message do not match the time from the code. Maybe the code should be adjusted to sleep only 0.2 seconds.

Reply:
Oh sorry, I m going to fix that. The time in the code is the needed one. 0.2 seconds is not enough time.

Reply:
Done, thanks!